draft-pauly-httpbis-geoip-hint.md

```diff
@@ -73,63 +95,99 @@ For example, the header for an entry "192.0.2.5,US,US-AL,Alabaster" would be:
     Sec-CH-IP-Geo = "US,US-AL,Alabaster"
 ~~~
 
-This field also defines a parameter, "feed", that contains the URI of the
-IP geolocation feed that is authoritative for this entry. For example:
-
-~~~
-    Sec-CH-IP-Geo = "SG,SG-01,Singapore"; feed="https://noc.ietf.org/geo/google.csv"
-~~~
-
-Servers that can provide different content based on Geohash hints SHOULD include
-the headers in their "Accept-CH" list.
+Given that the Sec-CH-IP-Geo is a high-entropy client hint, meaning it is
+a client hint that is not in the low-entropy hint table, the server
+MUST explicitly opt-in to receive the Geo Client Hint as defined in
+{{?RFC8942}}. It will not be sent by default and the server MAY
+indicate support for this hint via the Accept-CH header in the
+initial response:
 
 ~~~
     Accept-CH: Sec-CH-IP-Geo
 ~~~
 
-Servers also SHOULD indicate for any cacheable content if the IP geo hints will influence
-the cached content, using the "Vary" header.
+Servers SHOULD indicate for any cacheable content if the geo hints
+will influence the cached content, using the 'Vary' header. This will
+indicate that the server may have used this header as a determining
+factor when choosing a response:
 
 ~~~
     Vary: Sec-CH-IP-Geo
 ~~~
 
-# Server Behavior
+# Client Behavior
 
-Upon receiving a IP Geolocation Client Hint, a server can use the information to influence
-its behavior in various ways, such as determining the content of HTTP responses.
-
-Many servers have existing IP geolocation feeds that they use to identify client locations.
-Servers can choose to use the hint value in one of several ways:
-
-- Use the client hint information instead of consulting another geolocation feed.
-- Check the value of the "feed" parameter on the header and determine if it is a trusted feed.
-   If this feed is trusted, but is not the default feed used by the server, the server
-   can choose to prefer the feed indicated by the client.
-- Check the value of the "feed" parameter on the header and fetch a copy of the feed
-   to verify the mapping, if a copy of the feed has not been fetched recently.
-- If the feed indicated in the "feed" parameter is unknown or untrusted, but starts
-   becoming common, the server can flag this feed as one to be manually checked and
-   added, if appropriate. This allows servers to automatically discover when new
-   feeds and services are brought up..
-
-If the server is acting as a forward proxy, such as a CONNECT proxy, it can use the hint
-to determine an appropriate geo-mapped IP address to use for outbound connections, or a
-client subnet to present in the EDNS0 Client Subnet extension for DNS queries {{?RFC6891}}
-{{?RFC7871}}.
+If possible, the client SHOULD specify their geolocation. If location
+is not available, the client MAY send a default value or none at all.
```

Comment:
I don't see the value of a default value. We should either explain it or remove it